application-admintools-ui/src/main/resources/AdminTools/Code/Translations.xml

```diff
@@ -39,6 +39,20 @@
 admin.admintools.description=Configure Admin Tools Application
 AdminTools.Code.ConfigurationClass_serverLocation.hint=Path to the server location
 AdminTools.Code.ConfigurationClass_serverLocation=Server location
+AdminTools.dashboard.section.backend.title=Backend
```

Comment:
It should be `adminTools` as prefix. Above the upper case is used at the start because it's about the page name, but here it's just a module name